v1/samples/dotnet_core_dynamic_add_module/README.md

```diff
@@ -24,7 +24,7 @@ Building the sample (Linux)
 ---------------------------
 The sample code and dependancies gets build by running:
 ```
-tools\build.sh --enable-dotnet-core-binding --disable-native-remote-modules
+tools\build.sh --enable-dotnet-core-binding
```

Comment:
Drop the `--enable-dotnet-core-binding ` here?

Reply:
This sample (`dotnet_core_dynamic_add_module`) is specifically about the .NET Core binding, so I think it's correct as-is.